### PR TITLE
Add redux

### PR DESCRIPTION
**Pull request by kosher9**
- Added books actions.
- Added categories actions.
- Added store configuration.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const ADD = 'my-app/categories/ADD';
+const REMOVE = 'my-app/categories/REMOVE';
```
Comment (VICTOR-CHUKWU) [inline src/redux/categories/categories.js]: - [x] Acoording to project requirement, `categories.js` should have only an action`CHECK_STATUS` kindly remove this as it is not needed

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            ...category,
+            status: action.status,
+          };
+        }
+        return category;
+      });
+    default:
+      return state;
+  }
+}
```
Comment (VICTOR-CHUKWU) [inline src/redux/categories/categories.js]: - [x] Also kindly make your reducer to only have the `CHECK_STATUS` action type as it is only the action required according to the project requirement

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  };
+}
+
+export function remove(id) {
+  return {
+    type: REMOVE,
+    id,
+  };
+}
+
```
Comment (VICTOR-CHUKWU) [inline src/redux/categories/categories.js]: - [x] Kinldy remove these action functions as it is not needed in `category.js`

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @kosher9  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 



@kosher9  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] To make a professional readme please fix the broken link in the license section.
___

### Cheers and Happy coding! 💻 👏

aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**